index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+     <nav class="contain">
+      <a href="#" class="navbar-logo">John Doe</a>
+    <div class="humburger">
```

Comment:
Kindly wrap the hamburger in the `<nav>` tag 😃

- _The nav tag is for semantics reasons. Later in the project, you will add a navigation menu that opens when you click the hamburger._

Reply:
- [ ] @abreham1222 as previously requested kindly ensure your header is wrapped using the `header` tag for this section. Also, ensure the hamburger menu is wrapped using the `nav` tag.